brave-instrumentation/aws-java-sdk-v2-core/pom.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2016-2018 The OpenZipkin Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+    in compliance with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the License
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+    or implied. See the License for the specific language governing permissions and limitations under
+    the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>brave-instrumentation-parent</artifactId>
+    <groupId>io.zipkin.aws</groupId>
+    <version>0.14.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>brave-instrumentation-aws-java-sdk-v2-core</artifactId>
+
+  <properties>
+    <!-- SDK V2 requires Java 8 -->
+    <main.java.version>1.8</main.java.version>
+    <main.signature.artifact>java18</main.signature.artifact>
+
+    <main.basedir>${project.basedir}/../..</main.basedir>
+
+    <aws-sdk-java-v2.version>2.1.0</aws-sdk-java-v2.version>
```

Comment:
Maybe I should move this into the top level POM to make sure it doesn't get missed when we update deps?